### PR TITLE
planning: add ADR-0002 and post-MVP issue batch

### LINKED ISSUE #27: README Overhaul & System Walkthrough
# Issue: README Overhaul & System Walkthrough

## Summary
Rewrite `README.md` to clearly explain what the repository does end-to-end, including inputs, outputs, pipeline stages, and local validation commands.

## Why
Current onboarding and operational understanding are weaker than implementation quality. README should be the canonical contributor entrypoint.

## Scope
- Document repository purpose for humans and LLMs.
- Explain inputs: cached HTML, source mappings, config targets.
- Explain outputs: `data/chords.jsonl`, generated docs, generated SVGs.
- Document pipeline flow: ingest -> normalize -> generate -> validate.
- Add command reference and troubleshooting section.

## Acceptance Criteria
- New contributors can run full workflow using README only.
- README includes deterministic/provenance/legal guardrails.
- README commands match current `package.json` scripts.

## Validation Steps
```bash
npm run lint
npm test
npm run build
npm run validate
```

## Follow-ups
- Keep README updated whenever scripts or artifact paths change.

### PR DESCRIPTION
**Pull request by metal-gogo**
## Summary
- add ADR-0002 roadmap in planning/decisions
- add issue drafts 011-020 in planning/issues
- open and label GitHub issues #27-#36, then cross-link in draft files
- rewrite README with a full end-to-end project walkthrough (inputs, outputs, pipeline, module map, validation)

## Why
- establish a concrete prioritized post-MVP hardening plan
- document the repository clearly for contributors and operators

## Validation
- planning/documentation changes only (no runtime code changes)
- local checks run for docs update context as needed

## Related Issues
- Closes #27
- #28 #29 #30 #31 #32 #33 #34 #35 #36

### CODE REVIEW

File: planning/issues/016-parser-resilience-all-guitar-chords.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Keep behavior deterministic across runs.
+
+## Acceptance Criteria
+- Expanded parser tests pass across fixture matrix.
+- Failures are explicit and actionable.
+- No nondeterministic parsing behavior introduced.
+
+## Validation Steps
+```bash
+npm test -- --run test/unit/parser.allGuitarChords.test.ts
```
[the remark this answers is not in the document]
Comment (metal-gogo) [reply planning/issues/016-parser-resilience-all-guitar-chords.md]: Applied, thanks. Updated validation command syntax to `npm test -- <file>` in this draft.

File: planning/issues/017-normalization-edge-cases-and-id-stability.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Confirm deterministic ordering post-normalization.
+
+## Acceptance Criteria
+- Added tests cover key edge-case matrix.
+- Canonical IDs remain stable and compliant with ADR-0001.
+- No alias information is unintentionally dropped.
+
+## Validation Steps
+```bash
+npm test -- --run test/unit/normalize.test.ts
```
[the remark this answers is not in the document]
Comment (metal-gogo) [reply planning/issues/017-normalization-edge-cases-and-id-stability.md]: Applied, thanks. Updated validation command syntax to `npm test -- <file>` in this draft.
